lib/core/services/alfresco-api.service.ts

```diff
@@ -48,6 +48,8 @@ export class AlfrescoApiService {
 
     lastConfig: AlfrescoApiConfig;
 
+    private whitelistUrl: string[] = ['/activiti-app/api/enterprise/system/properties'];
```

Comment:
contextRootBpm| (Optional value that define the context Root of the Activiti API default value is activiti-app )|activiti-app |

activiti-app is not always present in all the configurations you need to get this param from contextRootBpm of the JS-API

Reply:
Plus can we call also this list exclude...